src/backend/marsha/core/api.py

```diff
@@ -317,16 +369,13 @@ def post(self, request):
             HttpResponse to reflect if the XAPI request failed or is successful
 
         """
-        lti_user = LTIUser(request.user)
+        user = request.user
+        lti_user = LTIUser(user)
         try:
-            video = Video.objects.get(pk=lti_user.video_id)
+            video = Video.objects.get(pk=user.id)
```

Comment:
It might work but it looks really wrong. We need to do something about this lib and the way it is using the request.user :scream:

Reply:
The problem is how we use the lib, not the lib itself IMO